_posts/plotly_js/maps/2019-08-12-plotly_js-map-index.html

```diff
@@ -0,0 +1,30 @@
+---
+permalink: javascript/mapbox-charts/
```

Comment:
This should be "maps" in order to agree with python and r
```suggestion
permalink: javascript/maps/
```